### ISSUE TITLE
Overwrites my repo, then can't pull after a force-push

### ISSUE THREAD

**Issue by wcerfgba** [GitJournal/GitJournal#473]
## Environment

Version: GitJournal 1.74.3+2301
Platform: android
isPro: false

**App version:**  GitJournal 1.74.3+2301
**Device information:**
 - OS name and version: Android 10
 - Manufacturer: Nokia
 - Model: 1 Plus, TA-1111

## Description

**Expected behavior:**
I have a repo already on GitHub. On adding the repo in GitJournal I expect it to pull all of the existing content.

**Current behavior:**
GitJournal added a new commit which deleted everything from the repo and added an empty `.gitignore`. When I fixed the repo history with a force-push, I try to pull in GitJournal but it does not update.

## Steps to reproduce

1. Connect to GitHub in app
2. Choose an existing repo from the GH repo list
3. App shows empty note list and tries to sync
4. After sync, app shows no notes
5. In repo in GitHub web UI, observe new commit with only empty `.gitignore` file

**Comment by wcerfgba** [comment]
Can confirm this is still misbehaving even after adding a `.gitignore` file: GitJournal creates a commit "Add gitignore file" which deletes all other files.

**Comment by a-jackson** [comment]
I'm getting the same as this on 1.75.0+2317.
It wasn't possible to revert the commit because Git Journal just merged it in again and anything that creates a merge conflict leaves it unable to pull.

I ended setting up the repo in Git Journal then going to my computer and doing
```
git pull --no-ff
git revert HEAD^2 --no-commit
git commit --amend 
git push
```

now Git Journal's initial commit is still there but the merge commit with the rest of the repo just takes the original parent's changes entirely.
Git Journal happily pulls this and all my notes are there.

This did not happen when I originally set up Git Journal on another device some months ago but I couldn't say what version that was.

**Comment by vHanda** [comment]
I've figured it out. I'll push an update today.

**Comment by vHanda** [comment]
I've released version `1.75.1` with the fix. On both iOS and Android.

**Comment by a-jackson** [comment]
Just updated and tried it again. Works great. Thanks for the quick turnaround
